inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.13/src/test/java/org/apache/inlong/sort/tests/Mysql2KafkaTest.java

```diff
@@ -165,7 +166,7 @@ public void testKafkaWithSqlFile() throws Exception {
                     "INSERT INTO test_input "
                             + "VALUES (1,'jacket','water resistent white wind breaker',0.2, null, null, null);");
             stat.execute(
-                    "INSERT INTO test_input VALUES (2,'scooter','Big 2-wheel scooter ',5.18, null, null, null);");
+                    "INSERT INTO test_input VALUES (2,'scooter','Big 2-wheel scooter',5.18, null, null, null);");
```

Comment:
the extra blank will influence the test result ?

Reply:
No, it has no influence fot the test result.But this extra blank space may cause misunderstandings.So, I deleted it.